quinn-proto/src/config/transport.rs

```diff
@@ -203,6 +204,12 @@ impl TransportConfig {
         self
     }
 
+    /// Specifies if all application data packets should be padded to current maximum UDP payload size.
```

Comment:
Do we want to pad all packets in the application data space, or just all packets which carry application data (e.g. not ACK-only)? The former is implemented, but the wording here is a little ambiguous as to what the intent is; please clarify.

Reply:
Either works for for us, so whichever you prefer. For now I described the weaker variant to allow for implementation flexibility going forward.

I'm not sure how to best exclude ack-only UDP datagrams from padding if you would prefer that.

Reply:
I'm happy with the current behavior because it's simpler. Padding ACK-only datagrams will of course increase bandwidth consumption, but whether that matters is an application/deployment-specific question.